src/bvals/comms/tag_map.cpp

```diff
@@ -45,13 +45,20 @@ void TagMap::AddMeshDataToMap(std::shared_ptr<MeshData<Real>> &md) {
 }
 
 void TagMap::ResolveMap() {
+  #ifdef MPI_PARALLEL
+  int flag;
+  void *max_tag; // largest supported MPI tag value
+  MPI_Comm_get_attr( MPI_COMM_WORLD, MPI_TAG_UB, &max_tag, &flag);
+  #endif
```

Comment:
Is there a reason we should 
- not use `PARTHENON_MPI_CHECK` for the MPI call
- not check for the result of `flag` before we continue to work with `max_tag` below?

Reply:
@gshipman I think we were waiting for you to address @pgrete 's question here... after that, from my perspective, it's ready for merge.

Reply:
@Yurlungur see latest commit. Should be good if checks pass. Thx